package.json

```diff
@@ -8,8 +8,8 @@
   "private": false,
   "scripts": {
     "build:env": "sudo apt-get install libzmq3-dev -y && sudo apt install libsocketcan-dev -y && sudo apt install pkg-config -y && sudo apt install python3-dev -y && sudo apt install python3-pip -y && sudo pip3 install numpy",
-    "csp:clone": "sudo rm -rf libcsp && git clone --depth 1 https://github.com/AlbertaSat/libcsp",
-    "csp:configure": "cd libcsp && python3 waf configure --with-os=posix --enable-can-socketcan --enable-rdp --enable-crc32 --enable-hmac --enable-xtea --with-loglevel=debug --enable-debug-timestamp --enable-python3-bindings --with-driver-usart=linux --enable-if-zmqhub --enable-examples",
+    "csp:clone": "sudo rm -rf libcsp && git clone --branch master https://github.com/Albertasat/libcsp.git",
```

Comment:
I wasn't sure why we had "--depth 1", and I'm not sure why we need "--branch master" either :-)

Reply:
I think the master branch option was a part of the weird depth thing. I'll remove it